src/KnowOrigin.ts

```diff
@@ -0,0 +1,38 @@
+import { ERC721, Transfer } from "../generated/templates/NftContract/ERC721"
+import { Nft, NftContract } from "../generated/schema";
+import { fetchName, fetchSymbol, handleTransfer } from "./mapping";
+import { KnowOrigin } from "../generated/KnowOrigin/KnowOrigin";
+import { KNOW_ORIGIN, ZERO_ADDRESS } from "./constants";
+import { Address, BigInt, log } from "@graphprotocol/graph-ts";
+
+export function handleTransferKnowOrigin(event: Transfer): void {
+  let address = event.address.toHexString();
+  if (NftContract.load(address) == null) {
+    let nftContract = new NftContract(address);
+    nftContract.name = fetchName(event.address);
+    nftContract.symbol = fetchSymbol(event.address);
+    nftContract.platform = "KnowOrigin";
+    nftContract.save();
+  }
+    
+  handleTransfer(event);
+  
+  let id = event.transaction.hash.toHex() + "-" + event.logIndex.toString()
```

Comment:
We are using "/" in handleTransfer function, could you please use the same separator? `Nft.load(id)` on line 22 won't find corresponding NFT in the current implementation